templates/resources.html

```diff
@@ -157,7 +157,8 @@ <h3 class="p-heading--4">
       </div>
       <div class="col-4 u-align--center">
         <img
-          src="https://assets.ubuntu.com/v1/196ebe18-MAAS+Kubernetes.svg"
+          src="https://assets.ubuntu.com/v1/a9954c8e-hp-moonshot-server.jpg"
+          alt="MAAS and Kubernetes are a great combination for bare metal K8s."
```

Comment:
Thanks for fixing the validation of the page but this alt content adds little value to the adjacent content.Would only be read out by a screen reader. Can you leave this as an empty string please.

Reply:
I agree, similar to the previous one https://github.com/canonical-web-and-design/maas.io/pull/662#discussion_r769365979

Reply:
@caldav  can you please comment? you advised me to add things that would help with SEO.